src/aleph_client/commands/instance/__init__.py

```diff
@@ -770,16 +842,60 @@ async def confidential_start(
     valid = await client.validate_measure(sev_data, tik_path, firmware_hash=firmware_hash)
     if not valid:
         echo("Could not validate authenticity of the VM. Please check that you are validating against the proper hash")
-        return 2
+        return 1
     echo("Measurement are authentic")
 
-    secret_key = Prompt.ask(
-        "Please enter secret to start the VM",
-    )
+    secret_key = vm_secret or Prompt.ask("Please enter secret to start the VM", password=True)
 
     encoded_packet_header, encoded_secret = await client.build_secret(tek_path, tik_path, sev_data, secret_key)
     await client.inject_secret(vm_hash, encoded_packet_header, encoded_secret)
-    echo("Starting the instance...")
-    echo("Logs can be fetched using the `aleph instance logs` command")
-    echo("Run the following command to get the IPv6 address of your instance:  aleph instance list")
     await client.close()
+    console = Console()
```

Comment:
I'm not sure when should we use echo() and when we should use console. how do you decide?

Reply:
This dependency is already used on the page and it allows us to have colorful outputs